_commands/os_x/open.md

```diff
@@ -35,11 +31,35 @@ open -a Safari http://www.google.com
 open -R diary.txt
 ~~~
 
-####Additional Notes
+|     Flag     |               Description                |
+| :----------: | :--------------------------------------: |
+|      -a      |  Opens item with a specific application  |
+|      -e      |         Opens item with TextEdit         |
+|      -t      |   Opens item with default text editor    |
+|      -f      | Reads input from standard input and opens it with TextEdit |
+| -R, --reveal | Selects in the Finder instead of opening |
+|  -n, --new   | Opens a new instance of the application (even if it's already running) |
+|  -j, --hide  |         Launches the app hidden          |
+
 
-As we can see, open is fairly easy to use. For more flags, such as launching an application as hidden, please check out the man page.
+#### `open -a Safari --hide http://www.google.com`
+Opens google.com in Safari minimized by default
+##### Break it down
+The `-a` flag lets you choose an application to open your item with. Without it, the item would open up with the default handler. For example, if you remov `-a Safari` and your default browser is Chrome then google.com will open in Chrome.
```

Comment:
This is what this looks like: 
<img width="973" alt="screen shot 2016-03-19 at 3 14 31 pm" src="https://cloud.githubusercontent.com/assets/14142637/13900820/769f6e02-ede5-11e5-8791-c4c5f3ce3d3e.png">

Fix formatting.